stdlib/transactions/src/main/ballerina/src/transactions/transaction_block.bal

```diff
@@ -364,30 +410,68 @@ function registerLocalParticipant(string transactionBlockId, function (string tr
+                                        function (string trxId) abortedFunc) returns  TransactionContext? {
+    return externRegisterRemoteParticipant(java:fromString(transactionBlockId), committedFunc, abortedFunc);
+}
+
+function externRegisterRemoteParticipant(handle transactionBlockId, function (string trxId) committedFunc,
+                    function (string trxId) abortedFunc) returns TransactionContext? = @java:Method {
+    class: "io.ballerina.transactions.Utils",
+    name: "registerRemoteParticipant"
+} external;
 
 # Notify the transaction resource manager on local participant failture.
-function notifyLocalParticipantOnFailure() = external;
+function notifyLocalParticipantOnFailure() = @java:Method {
+    class: "io.ballerina.transactions.Utils",
+    name: "notifyLocalParticipantOnFailure"
+} external;
 
 # Notify the transaction resource manager on remote participant failture.
-function notifyRemoteParticipantOnFailure() = external;
+function notifyRemoteParticipantOnFailure() = @java:Method {
+    class: "io.ballerina.transactions.Utils",
+    name: "notifyRemoteParticipantOnFailure"
+} external;
 
 # Notify the transaction resource manager on abort.
 #
 # + transactionBlockId - ID of the transaction block.
-function notifyResourceManagerOnAbort(string transactionBlockId) = external;
+function notifyResourceManagerOnAbort(string transactionBlockId) {
+    externNotifyResourceManagerOnAbort(java:fromString(transactionBlockId));
+}
+
+function externNotifyResourceManagerOnAbort(handle transactionBlockId) = @java:Method {
+    class: "io.ballerina.transactions.Utils",
+    name: "notifyResourceManagerOnAbort"
+} external;
 
 # Rollback the transaction.
 #
 # + transactionBlockId - ID of the transaction block.
-function rollbackTransaction(string transactionBlockId) = external;
+function rollbackTransaction(string transactionBlockId) {
+    externRollbackTransaction(java:fromString(transactionBlockId));
+}
+
+function externRollbackTransaction(handle transactionBlockId) = @java:Method {
+    class: "io.ballerina.transactions.Utils",
+    name: "rollbackTransaction"
+} external;
 
 # Cleanup the transaction context.
 #
 # + transactionBlockId - ID of the transaction block.
-function cleanupTransactionContext(string transactionBlockId) = external;
+function cleanupTransactionContext(string transactionBlockId) {
+    externCleanupTransactionContext(java:fromString(transactionBlockId));
+}
+
+function externCleanupTransactionContext(handle transactionBlockId) = @java:Method {
+    class: "io.ballerina.transactions.Utils",
+    name: "cleanupTransactionContext"
+} external;
 
 # Get and Cleanup the failure.
 #
 # + return - is failed.
-function getAndClearFailure() returns boolean = external;
\ No newline at end of file
+function getAndClearFailure() returns boolean = @java:Method {
+    class: "io.ballerina.transactions.Utils",
+    name: "getAndClearFailure"
+} external;
```

Comment:
```suggestion
} external;

```

Reply:
This might be an issue with GitHub. Faced the same in a recent code review, where Github does not show the empty line at the end of the file.